README.md

```diff
@@ -32,10 +32,18 @@ you don't need to remember the addresses.
 
 ### Command Line
 
+To print the tags and their values, simply call the library with the PLC IP address and 
+the tags file.
 ```
 $ productivity the-plc-ip-address path/to/tags.csv
 ```
 
+Set values on the PLC from the command line using the --set flag with a string of valid
```

Comment:
I like the API but leading with "valid YAML" sounds complicated. How about:

---

Use `--set` to set values on the PLC:

```
$ productivity the-plc-ip-address path/to/tags.csv --set "{valve: false, setpoint: 4.45, label: foo}"
```

The `--set` parameter takes [YAML](https://en.wikipedia.org/wiki/YAML), a simple data format that allows you to easily set multiple tags at once.

---

The idea here is to allow the user to pattern-match for themselves before being introduced to new terminology (which could take them down an unnecessarily complicated path). This version also gives a soft intro, which might provide enough explanation for users to avoid going down a googling path.